.github/ISSUE_TEMPLATE/1_bug_report.md

```diff
@@ -25,24 +25,24 @@ Include screenshots here if relevant.
 ***
```

Comment:
Should probably hide the line too?

Reply:
I did leave the titles as visible though to show they're not filled in. The hope being that they get filled out, rather than not done at all.

Reply:
But if they are not applicable, they go against @davidcmoulton's will not to have to delete them. For example `Related` is seldom necessary. `Tasks` may be made mandatory in this way.

Reply:
Agree that we could hide ones that are not mandatory (but disagree on how common 'Related' should be).